dev-ml/yojson/yojson-1.7.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit dune
+
+DESCRIPTION="JSON parsing and pretty-printing library for OCaml"
+HOMEPAGE="https://github.com/ocaml-community/yojson"
+SRC_URI="https://github.com/ocaml-community/yojson/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="examples +ocamlopt"
+
+RDEPEND=">=dev-lang/ocaml-4.02.3:=[ocamlopt]
+	dev-ml/easy-format:=[ocamlopt]
+	>=dev-ml/biniou-1.2:=[ocamlopt]
```

Comment:
We need to use 'ocamlopt=' to be [aligned](https://devmanual.gentoo.org/general-concepts/dependencies/) with whatever the USE flag applied is, otherwise it's constant/independent of what the user chooses.